### ISSUE TITLE
Is it RAD Tool or Power Tools?

### ISSUE THREAD

**Issue by SteveArr** [aspnetzero/documents#66]
Your documentation refers to the same tool using two totally different names.  You need to decide what the  name is and I will update the documentation to be consistent.

**Comment by hikalkan** [comment]
You're right. "Power tools" naming is better for a VS extension. But it's purpose is RAD. We will think about that. Thanks.

**Comment by hikalkan** [comment]
It should be "Power Tools" everywhere. "Rapid Application Development Tool" is just a description.
